app/lib/widgets/phone_widget.dart

```diff
@@ -238,21 +239,22 @@ class PhoneAlertDialogState extends State<PhoneAlertDialog> {
       Globals().tooManySmsAttempts = true;
       Globals().lockedSmsUntil = currentTime + 60000;
 
+      if (!mounted) return;
       showDialog(
         context: context,
-        builder: (BuildContext context) {
-          return AlertDialog(
-            title: const Text('Too many attempts please wait one minute.'),
-            actions: <Widget>[
-              TextButton(
-                child: const Text('OK'),
-                onPressed: () {
-                  Navigator.of(context).pop();
-                },
-              ),
-            ],
-          );
-        },
+        builder: (BuildContext childContext) => CustomDialog(
+          image: Icons.info,
+          title: 'Too many attempts',
+          description: 'Please wait one minute',
+          actions: <Widget>[
+            TextButton(
+              child: const Text('Ok'),
+              onPressed: () {
+                Navigator.pop(childContext);
+              },
+            ),
+          ],
+        ),
       );
       return;
     }
```

Comment:
Sending SMS already has a 2 min cool down. why do why have to add one min again here?

Reply:
- I didn't change the verification behavior itself, just adder counter and updated dialog style in phone widget.
- I believe the issue not in the scope of the pr.